README.md

```diff
@@ -79,7 +82,7 @@ This example uses CMake. Make sure that CMake and your build system of choice (e
 
 ### esptool.py
 
-To flash binaries into the ESP32-C3, [esptool.py](https://github.com/espressif/esptool) is used.
+To flash binaries into the ESP32-C3/ESP32-H2, [esptool.py](https://github.com/espressif/esptool) is used.
```

Comment:
Given that direct boot can also be supported on other chips, perhaps we could rewrite this to avoid having to keep expanding the list. For example,
```suggestion
To flash binaries into the chip, [esptool.py](https://github.com/espressif/esptool) is used.
```

Reply:
Agreed.
In that case it is better to name the target "chip" everywhere and then provide supported chips list.